README.md

```diff
@@ -30,14 +30,13 @@ It is intended for use with editor/IDE integration.
 - [defercheck](https://github.com/opennota/check) - Checks for duplicate defer calls.
 - [varcheck](https://github.com/opennota/check) - Find unused global variables and constants.
 - [structcheck](https://github.com/opennota/check) - Find unused struct fields.
-- [gotype](https://godoc.org/golang.org/x/tools/cmd/gotype) - Show compilation errors.
 - [errcheck](https://github.com/alecthomas/errcheck) - Check that error return values are used.
 - [dupl](https://github.com/mibk/dupl) - Reports potentially duplicated code.
 
 Disabled by default (enable with `--enable=<linter>`):
 
 - [testify](https://github.com/stretchr/testify) - Show location of failed testify assertions (disabled by default).
-- [test](http://golang.org/pkg/testing/) - Show location of test failures from the stdlib testing module (disabled by default).
```

Comment:
This is the canonical link. Stripping the trailing / just redirects and adds the / back.